tests/e2e/generate.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set +x
```

Comment:
Drop this.

Reply:
I am curious, why?

`set +x` disables a mode where all executed commands are printed.